git/cmd.py

```diff
@@ -629,7 +636,13 @@ def execute(self, command,
 
         def _kill_process(pid):
             """ Callback method to kill a process. """
-            p = Popen(['ps', '--ppid', str(pid)], stdout=PIPE)
+            if sys.platform == 'win32':
+                CREATE_NO_WINDOW = 0x08000000
+                creationflags = CREATE_NO_WINDOW
+            else:
+                creationflags = None
+
+            p = Popen(['ps', '--ppid', str(pid)], stdout=PIPE, creationflags)
```

Comment:
This concern is reflected in actual breakage as well: https://travis-ci.org/gitpython-developers/GitPython/jobs/148304657#L318